Packs/MicrosoftDefenderAdvancedThreatProtection/ReleaseNotes/1_17_0.md

```diff
@@ -0,0 +1,13 @@
+
+#### Integrations
+
+##### Microsoft Defender for Endpoint
+
+Add File indicator to the ***microsoft-atp-get-file-statistics*** command and fill the new indicator fields created under the CommonTypes pack:
```

Comment:
```suggestion
The ***microsoft-atp-get-file-statistics*** command now returns an indicator with the following details:
```

Reply:
Fixed in commit 18aefe6f94ba59b7039437d4b010a31d29cf5183